app/code/Magento/Indexer/Test/Mftf/ActionGroup/IndexerActionGroup/UpdateIndexerOnSaveActionGroup.xml

```diff
@@ -7,7 +7,7 @@
 -->
 <actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
-    <actionGroup name="updateIndexerOnSave">
+    <actionGroup name="UpdateIndexerOnSave">
```

Comment:
Replace that to `AdminIndexerSetUpdateOnSaveActionGroup`, also create `ActionGroup/_Deprecated_/updateIndexerOnSaveActionGroup.xml` with contents:
```xml
<actionGroup name="updateIndexerOnSave" extends="AdminIndexerSetUpdateOnSaveActionGroup" deprecated="Use AdminIndexerSetUpdateOnSaveActionGroup"/>
```
Please also keep in mind that filename has to be the same name that Action Group's name inside.